### PR TITLE
Updating the User Resource

### PR DESCRIPTION
**Pull request by dino-saurabh**
1. Added listing examples and response

### CODE REVIEW

**Comment by dino-saurabh** [comment]
@Akayeshmantha  please review !!

File: source/includes/User/user.md [hunk 1]
```diff
@@ -8,36 +8,122 @@ The real-life person is represented by a Person record in OpenMRS, and a person
...
-  .addHeader("Cookie", "JSESSIONID=154692F02BBBC664825F3C4C224A474B")
+  .addHeader("Cookie", "JSESSIONID=A2AF09658C73E1ECEC5D3C8C7C249A2D")
   .build();
 Response response = client.newCall(request).execute();
 
 ```
 
+```javascript
+
+var myHeaders = new Headers();
```
Comment (Akayeshmantha) [inline source/includes/User/user.md]: change this pls.

File: source/includes/User/user.md [hunk 2]
```diff
@@ -67,68 +153,126 @@ Request request = new Request.Builder()
...
 Response response = client.newCall(request).execute();
 
 ```
 
-  Retrieve a user by its UUID. Returns a `404 Not Found` status if the user does not exist. If not logged in to perform this action, a `401 Unauthorized` status is returned.
+```javascript
+
+1. Here the target UUID used is of the admin user.  
+
+var myHeaders = new Headers();
```
Comment (Akayeshmantha) [inline source/includes/User/user.md]: ditto

File: source/includes/User/user.md [hunk 3]
```diff
@@ -235,6 +408,24 @@ Response response = client.newCall(request).execute();
 
 ```
 
+```javascript
+
+var myHeaders = new Headers();
+myHeaders.append("Authorization", "Basic YWRtaW46QWRtaW4xMjM=");
```
Comment (Akayeshmantha) [inline source/includes/User/user.md]: ditto

**Review by Akayeshmantha — CHANGES_REQUESTED** [review]

**Comment by LuGO0** [comment]
Sure @Akayeshmantha I will make the changes 👍

File: source/includes/User/user.md (hunk 1 above)
Comment (LuGO0) [reply source/includes/User/user.md:109]: @Akayeshmantha I dint get should I remove the empty line ?

**Comment by dino-saurabh** [comment]
@Akayeshmantha updated with changes

**Comment by Akayeshmantha** [comment]
LGTM

**Pull request merged**